content/en/docs/reference/setup-tools/kubeadm/kubeadm-upgrade.md

```diff
@@ -8,25 +8,17 @@ content_template: templates/concept
 weight: 40
 ---
 {{% capture overview %}}
-`kubeadm upgrade` is a user-friendly command that wraps complex upgrading logic behind one command, with support
-for both planning an upgrade and actually performing it. `kubeadm upgrade` can also be used for downgrading
-cluster if necessary.
+`kubeadm upgrade` is a user-friendly command that wraps complex upgrading logic
+behind one command, with support for both planning an upgrade and actually performing it.
 {{% /capture %}}
 
 {{% capture body %}}
 
 ## kubeadm upgrade guidance
 
-Every upgrade process might be a bit different, so we've documented each minor upgrade process individually.
-For more version-specific upgrade guidance, see the following resources:
+The steps for performing a upgrade using kubeadm are outlined in [this document](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/).
 
- * [1.12 to 1.13 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13/)
- * [1.13 to 1.14 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-14/)
- * [1.14 to 1.15 upgrades](/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15/)
-
-_For older versions, please refer to older documentation sets on the Kubernetes website._
-
-In Kubernetes v1.11.0 and later, you can use `kubeadm upgrade diff` to see the changes that would be
+You can use `kubeadm upgrade diff` to see the changes that would be
```

Comment:
Please preserve:
_For older versions, please refer to older documentation sets on the Kubernetes website._